src/index.js

```diff
@@ -279,8 +281,8 @@ const SitumPlugin = {
     RNCSitumPlugin.removeNavigationUpdates(callback || warning);
   },
 
-  invalidateCache: function (callback?: Function) {
```

Comment:
@situm-abarros need to change iOS method signature as well

From
```
RCT_EXPORT_METHOD(invalidateCache:(RCTResponseSenderBlock)callback)
{
    NSMutableDictionary *obj = [[NSMutableDictionary alloc] init];
    [[SITCommunicationManager sharedManager] clearCache];
    callback(@[obj.copy]);
}
```
TO

```
RCT_EXPORT_METHOD(invalidateCache)
{
    NSMutableDictionary *obj = [[NSMutableDictionary alloc] init];
    [[SITCommunicationManager sharedManager] clearCache];
}
```

Would you do that or should I update in next diff?

Reply:
Thanks for your help! I tried to add the changes you suggested. Can you have a look just in case? I'm still a React newbie and maybe it's better to double check.

Reply:
Everything looks good, but I am curious - why removed POI methods first and then undo the removal?

Reply:
While testing those methods, I was seeing a crash in Android, so I thought it would be best to remove them temporarily untill we fixed the bug. Later, with the help of a colleague, I was able to figure out the problem. It was a mistake on my end. After fixing it, I could confirm it was working properly on Android, so I restored the methods.
Sorry for the trouble. 😅

Reply:
Oh right 😅
If you find any problems/ambiguities, let me know - I'll be happy to help 👌